frontend/src/features/landing_page/components/PhotoGallery.tsx

```diff
@@ -1,37 +1,39 @@
 import SimpleButton, { ButtonVariant } from '@/components/Button/SimpleButton';
 import '@/components/Modals/ImageModal';
 import ImageModal from '@/components/Modals/ImageModal';
+import { getPublicFileList, getPublicImage } from '@/features/nextcloud_integration/api/getImages';
+import { ImageBlob } from '@/features/nextcloud_integration/components/ImageBlob';
+import { useQuery } from '@tanstack/react-query';
 import { useState } from 'react';
 
+async function getPublicImages(imagePaths: Array<string>, publicShareToken: string) {
+  return Promise.all(imagePaths.map((path) => getPublicImage(path, publicShareToken)));
+}
+
+function getImagesFromFileList(files: Array<string>) {
```

Comment:
Isn't this a filter and not a getter? Please add a comment. Also consider to put it to '@/features/nextcloud_integration/api/getImages'